src/twisted/runner/procmon.py

```diff
@@ -103,14 +128,26 @@ class ProcessMonitor(service.Service):
     def __init__(self, reactor=_reactor):
         self._reactor = reactor
 
-        self.processes = {}
+        self._processes = {}
         self.protocols = {}
         self.delay = {}
         self.timeStarted = {}
         self.murder = {}
         self.restart = {}
 
 
+    @deprecate.deprecatedProperty(incremental.Version("Twisted", 17, 10, 0))
```

Comment:
I don't know where it is documented, but I remember seeing something like a placeholder which is used instead of the version.
At release, there is a script which will replace it with the actual version in which it is released

Reply:
This should be updated to NEXT like the one on line 43?

Can you point me at the docs for this functionality so that I can add it to the wiki page?

Reply:
I think https://github.com/twisted/incremental#updating is the citation.

Reply:
Yup, fixing.